### PR TITLE
Add Alarm Modes

### PR DESCRIPTION
**Pull request by Pierre-Gilles**
### Pull Request check-list

To ensure your Pull Request can be accepted as fast as possible, make sure to review and check all of these items:

- [x] If your changes affects code, did your write the tests?
- [x] Are tests passing? (`npm test` on both front/server)
- [x] Is the linter passing? (`npm run eslint` on both front/server)
- [x] Did you run prettier? (`npm run prettier` on both front/server)
- [x] If you are adding a new features/services, did you run integration comparator? (`npm run compare-translations` on front)
- [x] Did you test this pull request in real life? With real devices? If this development is a big feature or a new service, we recommend that you provide a Docker image to the community ([french forum](https://community.gladysassistant.com/)/[english forum](https://en-community.gladysassistant.com/)) for testing before merging.
- [x] If your changes modify the API (REST or Node.js), did you modify the API documentation? (Documentation is based on comments in code)
- [ ] If you are adding a new features/services which needs explanation, did you modify the user documentation? See [the GitHub repo](https://github.com/GladysAssistant/v4-website) and the [website](https://gladysassistant.com).
- [ ] Did you add fake requests data for the demo mode (`front/src/config/demo.js`) so that the demo website is working without a backend? (if needed) See [https://demo.gladysassistant.com](https://demo.gladysassistant.com).

### Description of change

Screenshot 2023-10-23 at 19 18 00

### CODE REVIEW

File: front/src/actions/house.js [hunk 1]
```diff
@@ -64,6 +64,31 @@ function createActions(store) {
...
+            alarm_code: {
+              $set: code
+            }
+          }
+        }
+      });
+      store.setState(newState);
+    },
+    updateHouseDelayBeforeArming(state, delayBeforeArming, houseIndex) {
+      console.log(delayBeforeArming);
```
Comment (callemand) [inline front/src/actions/house.js]: I think you need to remove `console.log`

**Review by callemand — CHANGES_REQUESTED** [review]
This banner is pretty weird. I think we should add a box around

Screenshot 2023-10-24 at 10 18 25

**Comment by callemand** [comment]
On "fullscreen=force" mode, why the "Mode Tablette" appear. I think we cannot make modification in "fullscreen=force", no ?
Screenshot 2023-10-24 at 10 23 18

**Comment by callemand** [comment]
When I click on all button, My alarm is activated and desactived in the same time 

https://github.com/GladysAssistant/Gladys/assets/11317212/7a294543-54e8-48c2-bed8-100813a1066f

**Comment by callemand** [comment]
Missing the "0", I cannot use a password like "1909"
Screenshot 2023-10-24 at 10 30 45

**Comment by Pierre-Gilles** [comment]
@callemand Thanks for the review! 🙏 

I fixed the console.log, the zero missing on the keypad and added a box around the tablet mode configuration.

For this:

Screenshot 2023-10-26 at 10 02 09

It's completely normal, the "full color" state is on when the button is clickable, and if you are partially armed, you can either "fully arm", or "disarm".

> On "fullscreen=force" mode, why the "Mode Tablette" appear. I think we cannot make modification in "fullscreen=force", no ?

There is no link between the fullscreen=force and tablet mode. You need the tablet mode button to be visible to configure the house the tablet is currently in. You can have a tablet not in full screen=force & a fullscreen=force not in tablet :)

File: front/src/actions/house.js (hunk 1 above)
Comment (Pierre-Gilles) [reply front/src/actions/house.js]: done

File: front/src/config/i18n/fr.json [hunk 2]
```diff
@@ -172,7 +180,20 @@
       "conflictError": "Une maison avec le même non existe déjà.",
       "roomConflictError": "Une pièce avec le même non existe déjà.",
       "validationError": "Erreur de validation : Le nom d'une maison doit-être compris entre 1 et 40 caractères.",
-      "validationErrorRoom": "Erreur de validation : le nom d'une pièce doit être entre 1 et 40 caractères."
+      "validationErrorRoom": "Erreur de validation : le nom d'une pièce doit être entre 1 et 40 caractères.",
+      "alarmTitle": "Alarme",
+      "alarmDescription": "Si vous utilisez le mode alarme dans Gladys, vous pouvez configurer ici le code de désactivation de l'alarme, ainsi que délai avant le déclenchement de l'alarme. Ce délai sera appliqué lors d'un déclenchement manuel uniquement, pas dans les scènes.",
```
Comment (cicoub13) [inline front/src/config/i18n/fr.json]: ainsi que **le** délai

File: front/src/config/i18n/fr.json [hunk 3]
```diff
@@ -190,6 +211,14 @@
     "editDashboardDeleteButton": "Supprimer",
     "editDashboardSaveButton": "Sauvegarder",
     "editDashboardDeleteText": "Êtes-vous sûr de vouloir supprimer ce tableau de bord ?",
+    "toggleDefineTabletMode": "Mode tablette",
+    "closeDefineTabletMode": "Fermer",
+    "tabletMode": {
+      "description": "Le mode tablette sert à la fonctionnalité alarme. Si vous armez l'alarme, toutes les tablettes de la maison seront verrouillées et afficherons un clavier virtuel pour désactiver l'alarme.",
```
Comment (cicoub13) [inline front/src/config/i18n/fr.json]: afficheront

File: front/src/config/i18n/fr.json [hunk 4]
```diff
@@ -338,6 +368,19 @@
...
+        "armButton": "Armer",
+        "disarmButton": "Désarmer",
+        "partiallyArmedButton": "Armement",
+        "partiallyArmedButtonSecondLine": "partiel",
+        "panicButton": "Panique",
+        "editBoxNameLabel": "Nom du widget",
+        "editBoxNamePlaceholder": "Entrez le nom du widget",
+        "editHouseLabel": "Sélectionnez la maison qui est concernée pour l'activation/désactivation de l'alarme.",
+        "alarmStatusText": "Votre maison est ",
+        "alarmArming": "Votre maison est entrain d'être armée...",
```
Comment (cicoub13) [inline front/src/config/i18n/fr.json]: en train

**Review by cicoub13 — CHANGES_REQUESTED** [review]
Some typos in FR

File: front/src/config/i18n/fr.json (hunk 4 above)
Comment (Pierre-Gilles) [reply front/src/config/i18n/fr.json]: Fixed

**Pull request merged**